fzf.el

```diff
@@ -244,6 +244,25 @@ If nil, fzf prompts have the same history in all modes."
   :safe #'booleanp
   :group 'fzf)
 
+(defcustom fzf/position-bottom-of-frame nil
+  "Set the position of the fzf to always be at the bottom of the frame."
+  :type 'boolean
+  :safe #'booleanp
+  :group 'fzf)
+
+(defun fzf/position-bottom-of-frame-watcher (symbol newval operation where)
```

Comment:
The function does not use the `symbol`, `operation` and `where` arguments.  The byte compiler will complain.  Rename these arguments with a prefix underscore to prevent the byte compiler warning, as in:  `(_symbol newval _operation _where)`

Reply:
Fixed in e3f88ee